### PR TITLE
[Tor2Web] Update and reactivate ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**
Should I split this ruleset? In this PR or a new one?

### CODE REVIEW

**Comment by Bisaloo** [comment]
Not sure what's going on here. This is maybe due to the unstability of those domains (?)

**Comment by Bisaloo** [comment]
I think that some of these services are using exit nodes that are being blocked by the hidden service. Any advice of better onion urls to use in the test?

**Comment by J0WI** [comment]
If it's impossible you can white list it.

**Comment by koops76** [comment]
@J0WI 
`facebookcorewwwi.onion`

File: src/chrome/content/rules/Onion.nu.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="Onion.nu" default_off="time out">
```
Comment (koops76) [inline src/chrome/content/rules/Onion.nu.xml]: `timeout`

**Comment by J0WI** [comment]
@Bisaloo would you still like to continue here?

**Comment by zoracon** [comment]
Closing due to no revisions made. Can reopen if interest for revision is indicated

**Pull request closed without merging**